misc/azure-vault-sensitive-var/validate.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+if [[ "$AZURE_SHAG_FAKE_SECRET" == "FAKE_NEWS" ]];
+then
+  echo "Loaded secret from azure vault successfully"
+  exit 0
+else
+  echo "Validation failed! Value of azure vault secret is $AZURE_SHAG_FAKE_SECRET"
```

Comment:
what it outputs if the secret doesn't exist?

Reply:
Maybe I misunderstood something here but from what I see it will write the message `Validation failed! Value of azure vault secret is <the value it got instead of FAKE_NEWS>`.

Reply:
can it be something else if it failed retrieving the value?? my point, if the value is empty, don't keep it `secret is <blank>`, have something like `missing value`

Reply:
if the value exists but different, state what you expect and what was received, like now, just add the FAKE_NEWS there